client/js/color.js

```diff
@@ -1,3 +1,16 @@
+export function toHex(inputColor) {
+  const hexPattern = /^#([A-Fa-f0-9]{6}|[A-Fa-f0-9]{3})$/;
```

Comment:
#rgb should be expanded to #rrggbb (or just not match this case) as other code expects a full 6 hex digits.

Reply:
Done